src/android/ScreenshotSaver.java

```diff
@@ -1,5 +1,6 @@
 package com.darktalker.cordova.screenshot;
 
+import static com.xtreme.plugins.XtremePushPlugin.TAG;
```

Comment:
Could you please clarify, why do we need this line? and how does XtremePushPlugin connected to it?

Shouldn't we just define Tag here as smth like:
```
public class ScreenshotSaver {
  private static final String TAG = "ScreenshotSaver";
```